common.gradle

```diff
@@ -30,6 +30,8 @@ android {
         checkDependencies true
         checkReleaseBuilds false
         warningsAsErrors true
+
+        disable 'GradleDependency'
```

Comment:
We shouldn't need to keep our deps up to date with the latest/greatest. Especially when we want to compile/test against older versions of the IMA/GMA plugins.